common/lib/xmodule/xmodule/modulestore/mongo/base.py

```diff
@@ -1914,21 +1914,25 @@ def ensure_indexes(self):
         """
 
         # Because we often query for some subset of the id, we define this index:
-        self.collection.create_index([
-            ('_id.org', pymongo.ASCENDING),
-            ('_id.course', pymongo.ASCENDING),
-            ('_id.category', pymongo.ASCENDING),
-            ('_id.name', pymongo.ASCENDING),
-        ])
+        self.collection.create_index(
+            [
+                ('_id.tag', pymongo.ASCENDING),
```

Comment:
I think we only have 1 tag in our actual data. How will that impact the performance of this index?